src/huntsman/pocs/states/huntsman/dithering.py

```diff
@@ -16,7 +16,6 @@ def on_enter(event_data):
 
         if pocs.observatory.mount.set_target_coordinates(current_observation.field):
             pocs.observatory.slew_to_target()
-            pocs.status()
```

Comment:
What's the reason for removing these?

Reply:
@wtgee I'm not sure why they are there in the first place? And currently they are being called, but `pocs.status` returns a `dict`.